tools/image-builder/Makefile

```diff
@@ -87,11 +88,17 @@ builder_stamp:
 
 clean:
 	-rm -f *stamp
-	-rm -rf $(WORKDIR)
-	-rm -f rootfs.img
+	if [ -d $(WORKDIR) ] || [ -e rootfs.img ]; then \
+	  if [ $(UID) -eq 0 ]; then \
+	    rm -rf $(WORKDIR) \
+	    rm -f rootfs.img ;\
+	   else \
+	     $(MAKE) clean-in-docker ;\
+	   fi ; \
+	fi
 
-distclean: clean-in-docker
+distclean: clean
 	rm -rf files_ephemeral
-	docker rmi fc-image-builder:latest
+	- docker rmi fc-image-builder:latest
```

Comment:
If the `latest` tag is hardcoded, then I think parallel runs of the CI system will end up referencing the same image and probably interfere with each other. 

There an env var `DOCKER_IMAGE_TAG` that is set in in buildkite runs to a unique ID per-run, so I think we want to use that for the image-builder too now that it's being used in the CI.

Reply:
Good point, thanks. Will fix.

Reply:
Fixed.